src/components/Tokens/TokenDetails/BalanceSummary.tsx

```diff
@@ -49,16 +45,18 @@ const TotalBalanceItem = styled.div`
   display: flex;
 `
 
-export default function BalanceSummary({ address }: { address: string }) {
+export default function BalanceSummary({
+  address,
+  balanceNumber,
+  balanceUsdNumber,
+}: {
+  address: string
+  balanceNumber?: number
```

Comment:
nit: can you name this `balance` and usd would be `balanceValueUsd`

Reply:
done